src/components/core/core.js

```diff
@@ -203,6 +204,17 @@ export default class Core extends UIObject {
     this.mediaControl.show()
   }
 
+  handleWindowResize(event) {
+    let orientation = ($(window).width() > $(window).height()) ? 'landscape' : 'portrait'
+    if (this._screenOrientation === orientation) return
+    this._screenOrientation = orientation
+
+    this.trigger(Events.CORE_SCREEN_ORIENTATION_CHANGED, {
+      event: event,
+      orientation: this._screenOrientation
```

Comment:
I'm in favor of using an `Object` here even if it has a single element now `{orientation: 'x'}` and then change it also in doc (Event) because it leaves room for new attributes, what do you think?